docs/Configuration.md

```diff
@@ -35,6 +32,8 @@ up uxp install
 Find more information in the [Upbound UXP
 documentation](https://docs.upbound.io/uxp/).
 
+_The Upbound AWS official provider may also be used with upstream [Crossplane](https://crossplane.io/docs/v1.9/getting-started/install-configure.html)._
```

Comment:
```suggestion
_The Upbound AWS official provider may also be used with upstream [Crossplane](https://crossplane.io/docs/v1.10/getting-started/install-configure.html)._
```

Assuming there is no way to link 'latest' doc.

Reply:
```suggestion
_The Upbound AWS official provider may also be used with upstream [Crossplane](https://crossplane.io/docs/master/getting-started/install-configure.html)._
```

Reply:
I'll link to master that way its always the latest.